cartoframes/data/enrichment/polygons_enrichment.py

```diff
@@ -133,31 +133,33 @@ def enrich_polygons(data, variables, agg_operators, data_geom_column='geometry',
 
 
 def _prepare_sql(enrichment_id, filters_processed, table_to_geotable, table_to_variables,
-                 table_to_project, table_to_dataset, user_dataset, working_project, data_table, **kwargs):
+                 table_to_project, table_to_dataset, user_dataset, working_project,
+                 data_table, **kwargs):
 
     grouper = 'group by data_table.{enrichment_id}'.format(enrichment_id=enrichment_id)
 
     sqls = list()
 
     for table, variables in table_to_variables.items():
+        agg_operators = kwargs.get('agg_operators')
 
-        if 'agg_operators' in kwargs:
+        if agg_operators is not None:
```

Comment:
`agg_operators` must be None by default, otherwise we get an error cause the parameter is missing in, for instance, this code taken from the examples:

```py
dataset_enrich = enrichment.enrich_polygons(dataset, variables)
```

Reply:
You are right! Fixed!